lib/packageModules.js

```diff
@@ -26,7 +26,82 @@ function setArtifactPath(funcName, func, artifactPath) {
   }
 }
 
-function zip(directory, name) {
+/**
+ * Copy pasted from Serverless
+ *
+ * @see https://github.com/serverless/serverless/blob/63d54e1537e10ae63c171892edd886f6b81e83f6/lib/plugins/package/lib/zipService.js#L65
+ */
+function serverlessZip(args) {
```

Comment:
It seems the code coverage dropped because there are no unit tests for this new function.

Reply:
It's hard to test unit it, in fact, we would like to provide e2e tests which will more easy to cover that code.